src/xmrEngine.js

```diff
@@ -705,7 +705,13 @@ export class MoneroEngine {
       this.log.error(
         `broadcastTx failed: ${String(e)} ${cleanTxLogs(edgeTransaction)}`
       )
-      throw e
+      if (e.message.includes(' 422 ')) {
```

Comment:
It should be `if (e instanceof Error && e.message.includes(' 422 '))`, since it is possible to `throw` things that are not errors.